alpine/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM alpine
+RUN apk add --update curl python bash &&  rm -rf /var/cache/apk/*
```

Comment:
What about the `--no-cache` option to [skip caching altogether](https://github.com/gliderlabs/docker-alpine/blob/master/docs/usage.md#disabling-cache)?